xcmailr-webapp/src/main/java/conf/application.conf

```diff
@@ -162,25 +164,20 @@ ebean.ddl.run=false
 %test.ebean.ddl.run=true
 
 ebean.datasource.minConnections=1
-ebean.datasource.maxConnections=25
+ebean.datasource.maxConnections=50
 ebean.datasource.isolationlevel=read_committed
 ebean.datasource.heartbeatsql=select 1
 
-# uncomment the following line to enable stacktrace capturing used to detect connection leaks
-#eban.datasource.captureStackTrace=true
-%test.ebean.datasource.captureStackTrace=true
-%dev.ebean.datasource.captureStackTrace=true
-
```

Comment:
The capture stack trace properties seem useful to me. What about keeping them, maybe comment out?

Reply:
Done.